### ISSUE TITLE
DirectInputDevice.id_product_guid only used Data1 and neglects other GUID struct fields

### ISSUE THREAD

**Issue by wolke6000** [pyglet/pyglet#1050]
When the id_product_guid is set for DirectInputDevice ([here](https://github.com/pyglet/pyglet/blob/0581e2d17db7c9298e1e9dfeec3d1b3215c878e0/pyglet/input/win32/directinput.py#L76))...
```python
self.id_product_guid = format(device_instance.guidProduct.Data1, "08x")
```
...only `Data1` is used, but there are also Data2, Data3 and Data4

a more complete product GUID could be generated like so:

```python
i = device_instance.contents.guidProduct
guid = (f"{i.Data1:08x}-{i.Data2:04x}-{i.Data3:04x}-" +
           "".join(f"{x:02x}" for x in i.Data4[:2]) +
           "-" + "".join(f"{x:02x}" for x in i.Data4[2:]))
```

**Comment by benmoran56** [comment]
We can just expose the guidProduct struct directly, for those that are interested in it.

**Comment by wolke6000** [comment]
that would be satisfactory. ;-)

**Comment by benmoran56** [comment]
This will be included in the upcoming v2.1 release, which should start appearing as 2.1devX releases in the near future.
If you have an immediate need for this, please let me know.
